catalogue/src/main/kotlin/com/tidal/sdk/catalogue/Catalogue.kt

```diff
@@ -1,18 +1,81 @@
 package com.tidal.sdk.catalogue
 
-/**
- * Demo documentation block for [Catalogue]
- * @param optInt The constructor [Catalogue] takes an optional [Int]
- * just to tell you about in this comment.
- * As this is used directly by the demo application, which is using
- * Jetpack Compose, this is a data class,which makes it easily comparable
- * and helps prevent unnecessary recomposition
- */
-data class Catalogue(private val optInt: Int = 5) {
+import com.tidal.sdk.auth.CredentialsProvider
+import com.tidal.sdk.catalogue.di.DaggerCatalogueComponent
+import com.tidal.sdk.catalogue.generated.apis.AlbumJSONAPI
+import com.tidal.sdk.catalogue.generated.apis.ArtistJSONAPI
+import com.tidal.sdk.catalogue.generated.apis.ProviderJSONAPI
+import com.tidal.sdk.catalogue.generated.apis.TrackJSONAPI
+import com.tidal.sdk.catalogue.generated.apis.VideoJSONAPI
+import javax.inject.Inject
+import retrofit2.Retrofit
+
+class Catalogue private constructor() {
+
+    @Inject
+    lateinit var retrofit: Retrofit
+
+    /**
+     * Returns an instance of the [AlbumJSONAPI] which can be used
+     * to make API calls to the Album API
+     */
+    fun getAlbumAPI(): AlbumJSONAPI {
+        return retrofit.create(AlbumJSONAPI::class.java)
+    }
+
     /**
-     * This function returns the string "Catalogue! The string is: [optInt]"
+     * Returns an instance of the [ArtistJSONAPI] which can be used
+     * to make API calls to the Artist
+     * API
      */
-    fun helloCatalogue(): String {
-        return "Catalogue! The int is: $optInt"
+    fun getArtisAPI(): ArtistJSONAPI {
```

Comment:
```suggestion
    fun getArtistAPI(): ArtistJSONAPI {
```
Missing a `t` I assume?

Reply:
You passed the test! 🤣